### PR TITLE
Freezes requirements to TXT

### PR DESCRIPTION
**Pull request by arvind0598**
This fixes issue #49 as mentioned earlier.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,15 @@
-beautifulsoup4
-lxml
-requests
+astroid==2.3.1
```
Comment (svimanet) [inline requirements.txt]: It looks like you added all the modules you currently have installed in your py venv, but not many of these are acutally required in the repo. 
```
pip freeze | grep 'lxml\|requests\|beautifulsoup' >> requirements.txt
```

Comment (arvind0598) [reply requirements.txt]: I created the virtual environment right after cloning the project. I figure this is something along the lines of this question [here](https://stackoverflow.com/questions/30363976/how-should-i-generate-requirements-txt-pip-freeze-not-a-good-way)? Would it be advisable to remove the dependencies that were added?

Comment (svimanet) [reply requirements.txt]: They make great points in the thread, but some of the requirements listed here are not dependencies of the actual required packages. 

```
(bot_venv) svimanet > pip freeze
beautifulsoup4==4.8.1
certifi==2019.9.11
chardet==3.0.4
idna==2.8
lxml==4.4.1
requests==2.22.0
soupsieve==1.9.4
urllib3==1.25.6
(bot_venv) svimanet > pip show beautifulsoup4
Name: beautifulsoup4
Version: 4.8.1
Summary: Screen-scraping library
Home-page: http://www.crummy.com/software/BeautifulSoup/bs4/
Author: Leonard Richardson
Author-email: leonardr@segfault.org
License: MIT
Location: /home/svimanet/Github/chatbot/bot_venv/lib/python3.7/site-packages
Requires: soupsieve
Required-by: 
(bot_venv) svimanet > pip show lxml
Name: lxml
Version: 4.4.1
Summary: Powerful and Pythonic XML processing library combining libxml2/libxslt with the ElementTree API.
Home-page: http://lxml.de/
Author: lxml dev team
Author-email: lxml-dev@lxml.de
License: BSD
Location: /home/svimanet/Github/chatbot/bot_venv/lib/python3.7/site-packages
Requires: 
Required-by: 
(bot_venv) svimanet > pip show requests
Name: requests
Version: 2.22.0
Summary: Python HTTP for Humans.
Home-page: http://python-requests.org
Author: Kenneth Reitz
Author-email: me@kennethreitz.org
License: Apache 2.0
Location: /home/svimanet/Github/chatbot/bot_venv/lib/python3.7/site-packages
Requires: certifi, idna, chardet, urllib3
Required-by: 
```

Also created a new venv and installed req:
```
(env) svimanet > pip install -r requirements.txt
...
(env) svimanet > pip freeze
beautifulsoup4==4.8.1
certifi==2019.9.11
chardet==3.0.4
idna==2.8
lxml==4.4.1
requests==2.22.0
soupsieve==1.9.4
urllib3==1.25.6
```

Comment (arvind0598) [reply requirements.txt]: Ah, I figured out my issue. The other dependencies were from the pylint installation that vscode did. I'll fix that.

**Review by svimanet — APPROVED** [review]

**Pull request merged**